libarchive/archive_write_set_format_gnutar.c

```diff
@@ -504,7 +504,7 @@ archive_write_gnutar_header(struct archive_write *a,
 		archive_entry_set_uname(temp, "root");
 		archive_entry_set_gname(temp, "wheel");
 
-		archive_entry_set_pathname(temp, "././@LongLink");
+		archive_entry_set_pathname(temp, "././@LongName");
```

Comment:
The `L` header is a long _FILENAME_ header.  I don't know why this was `@LongLink` before.